app/src/main/java/org/mozilla/fenix/settings/TabsSettingsFragment.kt

```diff
@@ -104,12 +103,10 @@ class TabsSettingsFragment : PreferenceFragmentCompat() {
     }
 
     private fun sendTabViewTelemetry() {
-        val metrics = requireContext().components.analytics.metrics
-
         if (listRadioButton.isChecked && !gridRadioButton.isChecked) {
-            metrics.track(TabViewSettingChanged(Type.LIST))
+            Events.tabViewChanged.record(Events.TabViewChangedExtra("list"))
```

Comment:
Looking at the original code we seems to be reporting `type.toString()` which makes it "LIST" and "GRID" (all cap).  Can you please confirm?

Reply:
In the original `TabViewSettingChanged` a conversion was used:`type.toString().lowercase(Locale.ROOT)` so I tried to keep the original extra keys, even the letter case, although there was not a strict rule (uppercase or lowercase) enforced throughout the app.

Reply:
Ah i see, it is using lowercase.  Thanks for confirming.